kubernetes/kubectl/kubectl.go

```diff
@@ -0,0 +1,7 @@
+package kubectl
+
+import "github.com/giantswarm/clustertpr/kubernetes/kubectl/docker"
+
+type Kubectl struct {
+	Docker docker.Docker `json:"docker" yaml:"docker"`
```

Comment:
This is just for structure and readability from the client point of view. It makes the templates really nice and easy to understand.